Packs/DSPM/Integrations/DSPM/DSPM.py

```diff
@@ -0,0 +1,903 @@
+        "dspmTagKey.equals": args.get("dspmTagKeyEqual"),
+        "dspmTagValue.in": args.get("dspmTagValueIn"),
+        "dspmTagValue.equals": args.get("dspmTagValueEqual"),
+        "projectId.in": args.get("projectIdIn"),
+        "projectId.equals": args.get("projectIdEqual"),
+        "cloudProvider.in": args.get("cloudProviderIn"),
+        "cloudProvider.equals": args.get("cloudProviderEqual"),
+        "affects.in": args.get("affectsIn"),
+        "affects.equals": args.get("affectsEqual"),
+        "status.in": args.get("statusIn"),
+        "status.equals": args.get("statusEqual"),
+        "sort": args.get("sort"),
+        "page": page,
+        "size": MAX_PAGE_SIZE,
+    }
+    # Remove None values from params
+    params = {k: v for k, v in params.items() if v is not None}
+    demisto.debug(f"params : {params}")
+
+    # Fetch data from client
+    findings = client.fetch_risk_findings(params)
+
+    if not findings:
+        return []  # No more findings to fetch
+
+    parsed_findings = [
+        {
+            "ID": finding.get("id", ""),
+            "Rule Name": finding.get("ruleName", ""),
+            "Severity": finding.get("severity", ""),
+            "Asset Name": finding.get("asset", {}).get("name", ""),
+            "Asset ID": finding.get("asset", {}).get("assetId", ""),
+            "Status": finding.get("status", ""),
+            "Project ID": finding.get("projectId", ""),
+            "Cloud Provider": finding.get("cloudProvider", ""),
+            "Cloud Environment": finding.get("cloudEnvironment", ""),
+            "First Discovered": finding.get("firstDiscovered", ""),
+            "Compliance Standards": finding.get("complianceStandards", {}),
+        }
+        for finding in findings
+    ]
+    return parsed_findings
+
+
+def get_risk_finding_by_id(
+    client: Client, args: dict[str, Any]
+) -> CommandResults:
+    risk_id = args.get("finding_id")
+    if not risk_id:
+        raise ValueError("finding_id argument is required")
+
+    # Fetch data from client using the get_risk_information method
+    response = client.get_risk_information(risk_id)
+
+    if not response:
+        raise ValueError(f"No risk found with id {risk_id}")
+
+    finding = response if isinstance(response, dict) else response[0]
+
+    parsed_finding = {
+        "ID": finding.get("id", ""),
+        "Rule Name": finding.get("ruleName", ""),
+        "Severity": finding.get("severity", ""),
+        "Asset Name": finding.get("asset", {}).get("name", ""),
+        "Asset ID": finding.get("asset", {}).get("assetId", ""),
+        "Status": finding.get("status", ""),
+        "Project ID": finding.get("projectId", ""),
+        "Cloud Provider": finding.get("cloudProvider", ""),
+        "Cloud Environment": finding.get("cloudEnvironment", ""),
+        "First Discovered": finding.get("firstDiscovered", ""),
+        "Compliance Standards": finding.get("complianceStandards", {}),
+    }
+
+    return CommandResults(
```

Comment:
please add **readable_output**  with tableToMarkdown to **all the CommandResults in the integration**, see this articles:
https://xsoar.pan.dev/docs/integrations/code-conventions#commandresults
https://xsoar.pan.dev/docs/integrations/code-conventions#tabletomarkdown

if you use `headerTransform=pascalToSpace` it will replace in the table `firstDiscovered` to `First Discovered ` for example

Reply:
Alright, we will update CommandResults as you suggested for all commands.